app/views/posts/show.html.erb

```diff
@@ -1 +1,7 @@
-<p><%= @specific_post %></p>
\ No newline at end of file
+<p>Post# <%= @specific_post.id %> by <%= @user.name %></p>
+<p>Comments: <%= @specific_post.comments_counter%>, Likes: <% @specific_post.likes_counter %></p>
+<p><%= @specific_post.text %><p>
+<%@specific_post.comments.each do |comment|%>
+<%= comment.author.name %>, <%= comment.text %>
+<br>
+<%end%>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes), Please add basic styling to Posts show view to make it match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png).